src/main/java/org/toradocu/extractor/ThrowsTag.java

```diff
@@ -1,71 +1,43 @@
 package org.toradocu.extractor;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
 import java.util.Objects;
+import java.util.StringJoiner;
+import org.toradocu.translator.spec.ExcPostcondition;
 import org.toradocu.util.Checks;
 
 /**
  * This class represents a throws tag in a method. Each @throws tag consists of an exception, a
```

Comment:
Like in other classes, this comment leads to confusion between the abstraction and the thing being represented.